src/Patches/PlayerControllerPatch.cs

```diff
@@ -156,17 +187,21 @@ public static void PostPlayerHitGroundEffects(PlayerControllerB __instance)
                 return;
             }
 
-            __instance.playerBodyAnimator.SetTrigger("startCrouching");
-            __instance.playerBodyAnimator.SetBool("crouching", true);
+            __instance.playerBodyAnimator.SetTrigger(StartCrouching);
+            __instance.playerBodyAnimator.SetBool(Crouching, true);
         }
 
-        private static bool CanJump(PlayerControllerB __instance) => !Physics.Raycast(
-            __instance.gameplayCamera.transform.position,
-            Vector3.up,
-            out __instance.hit,
-            0.72f,
-            __instance.playersManager.collidersAndRoomMask,
-            QueryTriggerInteraction.Ignore
-        );
+        private static bool CanJump(PlayerControllerB __instance)
+        {
+            RaycastHit hit = (RaycastHit)AccessTools.Field(typeof(PlayerControllerB), "hit").GetValue(__instance);
+            return !Physics.Raycast(
+                __instance.gameplayCamera.transform.position,
+                Vector3.up,
+                out hit,
+                0.72f,
+                __instance.playersManager.collidersAndRoomMask,
+                QueryTriggerInteraction.Ignore
+            );
+        }
```

Comment:
The changes made here seem to be entirely unnecessary and possibly might've degraded performance due to `AccessTools` (reflection) being invoked every jump.